src/db.js

```
@@ -0,0 +1,19 @@
+'use strict';
+
+const { Sequelize } = require('sequelize');
+const utils = require('util');
+
+require('dotenv').config();
+global.TextEncoder = utils.TextEncoder;
+
+const { DB_HOST, DB_PORT, DB_USER, DB_PASSWORD, DB_NAME } = process.env;
+const sequelize = new Sequelize({
+  database: DB_NAME || 'postgres',
+  username: DB_USER || 'postgres',
+  host: DB_HOST || 'localhost',
+  dialect: 'postgres',
+  port: DB_PORT || 5432,
+  password: DB_PASSWORD || '123',
```

Using a hard-coded default password (`'123'`) is risky and can mask misconfiguration. Consider throwing an explicit error if required DB credentials are missing in non-development environments, or at least document that defaults exist only for local development. Prefer `password: DB_PASSWORD || undefined` or require the password in CI/production.